tpcx_bb/xbb_tools/utils.py

```diff
@@ -442,7 +396,7 @@ def get_query_number():
         with open("current_query_num.txt", "r") as file:
             QUERY_NUM = file.read()
```

Comment:
We might need to cast the query number to int here.

Reply:
Can you clarify why? The benchmark runner and individual queries run without casting to int.

The only side effect of not casting I've noticed is that a Google Sheet will include "01" instead of "1". That maybe a problem with our old complex filtering logic, but I'm not sure it will be a problem going forward. Thoughts @beckernick ?

Reply:
My bad, re-read the code we actually rely on treating them on strings right now.   I think we should stick to a single standard now, i.e either treat everywhere like `05` or `5`, I am fine with either. 

Having it both ways (like we had previously) is confusing. 

https://github.com/rapidsai/tpcx-bb/blob/aa288d0c22229e7ad265c486fc9082e563032ea2/tpcx_bb/xbb_tools/utils.py#L179-L180

Reply:
I believe this is entirely related to Google Sheets formatting, so it can be removed if we don't mind potentially awkward formatting

Reply:
Actually treating is as string will require the least code changes, I am fine with the code as is right now.  Will keep this conversation unresolved in case @beckernick wants to weigh in.

Reply:
Yes, we need to keep the actual result as "01" and "09" currently (or change it in the correctness check functions), but the actual int conversion is for google sheets only